src/views/PluginFlags.tsx

```diff
@@ -93,11 +27,18 @@ function warningText(t: TranslationFunction, key: string) {
 }
 
 const PluginFlags = (props: IProps): JSX.Element => {
-  const { plugin, gameMode, t } = props;
+  const { 
+    plugin,
+    gameMode,
+    t,
+    doesSupportsESL,
+    getMinRevision,
+    isGameSupported,
+  } = props;
 
   const flags: JSX.Element[] = [];
 
-  if (!gameSupported(gameMode)) {
```

Comment:
As discussed yesterday I would like to avoid unnecessary changes.
Renaming gameSupported -> isGameSupported, supportsESL -> doesSupportsESL, ... is purely preference. If everyone did this we'd just go around in circles renaming each others stuff

Reply:
Renaming reverted